src/main/scala/catslib/Monad.scala

```diff
@@ -138,4 +138,11 @@ object MonadSection extends FlatSpec with Matchers with org.scalaexercises.defin
 
     optionTMonad[List].pure(42) should be(OptionT(res0))
   }
+ /** There are also instances for other monads available for user in Cats library:
+   *  'EitherT' for 'Either'
+   *  'ReaderT' for 'Reader'
+   *  'WriterT' for 'Writer'
+   *  'StateT' for 'State'
+   *  'IdT' for 'Id'
+   */
```

Comment:
Hi! Thanks for the contribution. How will this look when formatted on screen? Can you attach a screen shot please?

Reply:
![image](https://user-images.githubusercontent.com/15636676/82717112-7719c000-9ca3-11ea-8707-50ac09863379.png)